.github/CONTRIBUTING.md

```diff
@@ -321,20 +322,46 @@ https://doc.qt.io/qt-5/i18n-source-translation.html#handling-plurals)
 
 If you're about to propose a change that adds or modifies any translatable
 string in the code, you don't need to take care of adding the new strings to
-the translation files.
-Every few days, or when a lot of new strings have been added, someone from the
-development team will take care of extracting all the new strings and adding
-them to the english translation files and making them available to translators
-on Transifex.
+the translation files.<br>
+We have an automted process setup to update our language source files accordingly
+and provide the translators on Transifex with the new contents.<br>
+Maintainers can also manually trigger this on demand.
 
 ### Maintaining Translations (for maintainers) ###
 
-When new translatable strings have been added to the code, a maintainer should
-make them available to translators on Transifex. Every few days, or when a lot
-of new strings have been added, a maintainer should take care of extracting all
-the new strings and add them to the english translation files.
+When new translatable strings have been added to the code, a maintainer has to
+make them available to translators on Transifex.
 
-To update the english translation files, re-run cmake enabling the appropriate
+To help with that, we have an automated CI run setup, that regularly looks at the
```

Comment:
github calls them workflows instead of CI run setup and I'd like to keep this a bit more consistent

Reply:
Yep. That makes sense.

`setup` should have been `set up`. Will just remove it.